apps/crn-server/test/data-providers/calendars.data-provider.test.ts

```diff
@@ -204,7 +207,7 @@ describe('Calendars data provider', () => {
     });
 
     describe('Active filter', () => {
-      test('Should skip the calendars which belong to an inactive group when active is set to true', async () => {
+      test('Should skip the calendars which belong to an inactive group and complete working group when active is set to true', async () => {
```

Comment:
to be fair this test was easy to read when it was just that one condition (ie skip calendars linked to an inactive group) but now its a bit of a nightmare. 

Plus - do you honestly thing with the extra requirement you have tested all the scenarios? I would add tests for when
1) Active flag is on - calendar belongs to an inactive group and a working group that is not complete - it shows
2) Active flag is on - calendar belong to an active group and a working group that is complete - it shows
3) Active flag is on - calendar belongs to an inactive group and a working group that is complete - it does not show - this is technically the test i am commenting in now - however maybe its better to run this scenario in isolation, meaning only one calendar in the input fixture
4) Active flag is on - multiple calendars, one belongs to an active group and a complete working group - it shows, second belongs to an inactive group and an incomplete working group - it shows, third belongs to an inactive group and a working group that is completed - it does not show, fourth does not belong to any groups or working groups - it shows
5) Active flag is on - calendar belongs to multiple groups of which one is inactive - it shows (this test already exists https://github.com/yldio/asap-hub/blob/14e72948d47f9b70ef1c1342cd9b732a232ea991/apps/crn-server/test/data-providers/calendars.data-provider.test.ts#L344)
6) Active flag is on - calendar belogns to multiple working groups of which one is completed - it shows
7) Active flag is off - same as 4 except this time all calendars are shown